ListDetail/src/main/java/com/microsoft/device/display/samples/listdetail/ui/view/ListView.kt

```diff
@@ -20,56 +20,52 @@ import androidx.compose.material.Scaffold
 import androidx.compose.material.Text
 import androidx.compose.material.TopAppBar
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.livedata.observeAsState
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.res.colorResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import com.microsoft.device.display.samples.listdetail.R
-import com.microsoft.device.display.samples.listdetail.models.AppStateViewModel
 import com.microsoft.device.display.samples.listdetail.models.images
 import com.microsoft.device.dualscreen.twopanelayout.navigateToPane2
 
 private val imagePadding = 10.dp
 private val verticalPadding = 35.dp
 private val horizontalPadding = 15.dp
-private val imageMargin = 1.dp
 
 @Composable
-fun ListViewWithTopBar(appStateViewModel: AppStateViewModel) {
+fun ListViewWithTopBar(isDualScreen: Boolean, selectedIndex: Int, updateSelectedIndex: (Int) -> Unit) {
     Scaffold(
-        topBar = {
-            TopAppBar(
-                title = {
-                    Text(
-                        text = stringResource(R.string.app_name),
-                        style = TextStyle(
-                            fontSize = 18.sp,
-                            fontWeight = FontWeight.Bold,
-                            color = Color.White
-                        )
-                    )
-                }
+        topBar = { ListViewTopBar() },
+        content = { ListView(isDualScreen, selectedIndex, updateSelectedIndex) }
+    )
+}
+
+@Composable
+fun ListViewTopBar() {
+    TopAppBar(
+        title = {
+            Text(
+                text = stringResource(R.string.app_name),
+                style = TextStyle(
+                    fontSize = 18.sp,
+                    fontWeight = FontWeight.Bold,
+                    color = Color.White
```

Comment:
I think this property can be deleted and we can just rely on the default settings - in `Theme.kt` if you change `onPrimary` to `Color.White`, I believe Compose will recognize that the TopAppBar is either the primary/surface color and will know to use white (and I think the weight/size are already very similar to the defaults for TopAppBar titles)

Reply:
Ah yes, in dark mode I see that happen! When I switch to light mode, though, it tries to use black on the primary color (which I don't think has enough contrast):
![image](https://user-images.githubusercontent.com/33138268/149529540-95998f03-4ae8-42e6-b417-b0271c6935f0.png)

We could also just change the top bar to be gray all the time, up to you!